phpstan/ForbiddenRamseyNamespaceRule.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\phpstan;
+
+use PhpParser\Node;
+use PhpParser\Node\Stmt\Use_;
+use PHPStan\Analyser\Scope;
+use PHPStan\Rules\Rule;
+
+class ForbiddenRamseyNamespaceRule implements Rule
+{
+    public function getNodeType(): string
+    {
+        return Use_::class;
+    }
+
+    public function processNode(Node $node, Scope $scope): array
+    {
+        foreach ($node->uses as $use) {
+            if (! str_contains($use->name->toString(), 'Ramsey')) {
+                continue;
+            }
+
+            $filePath = $scope->getFile();
+            if (str_ends_with($filePath, 'src/Model/ValueObject/Identity/Uuid.php')) {
```

Comment:
If the Uuid file is moved this will break. Is it not possible to add and ignore comment?

Reply:
good idea, lets see...

Reply:
it works!